lib/wibox/layout/mirror.lua

```diff
@@ -17,7 +17,9 @@ local mirror = { mt = {} }
 
 --- Draw this layout
 function mirror:draw(wibox, cr, width, height)
-    if not self.widget then return { width = 0, height = 0 } end
+    if not self.widget or not self.widget.visible then
+        return { width = 0, height = 0 }
+    end
```

Comment:
> oh

That means to not drop it?
Or would/should it get handled by `base.draw_widget` anyway, too?